addons/hr_contract/static/src/components/contract_chat/contract_chat.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-name="hr.contract.OpenChat">
+        <a  t-on-click.prevent="() => openChat(this.props.record.data.employee_id[0])"
+            class="ml8 o_employee_chat_btn"
+            role="button"
+            title="Chat"
+            icon="fa-comments">
+            <i class="fa fa-comments align-middle fs-6"/>
```

Comment:
I think it would look better centered with the activity icon without the `align-middle`

Reply:
Yep right it's more aligned with the clock